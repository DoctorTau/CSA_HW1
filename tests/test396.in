571
415 -328 3520 4467 3096 -6193 -2362 -5441 8181 -8246 6615 -5294 -1842 -9162 -309 2713 781 6849 2688 8486 -7349 -5679 7892 2340 824 5675 -9918 -3121 -8010 8052 -3553 -644 -2770 -7850 -6211 -7509 -1131 -5815 7035 -3055 -329 5084 1959 8133 -9054 -6685 2976 -925 -5558 5883 8941 -4877 -3342 1076 -8868 4609 8382 -3061 -6659 4534 -1886 2623 6474 4681 4788 -7809 9745 8487 983 -8833 8679 3611 8157 -3727 -6213 -3233 -9702 -474 1725 -7009 -5318 -943 1976 9153 -7243 9169 -9923 -3116 5425 -1237 9282 7279 1262 -3531 -5110 -49 -8508 457 5326 2639 -1010 7669 5470 -3112 4777 -2109 -7389 -6385 -6549 8287 -2411 -9741 324 1706 7809 700 -9536 -2468 -8062 8612 -1749 9593 -2030 -6920 7971 1858 -644 1983 -2819 2258 282 -7832 -7934 4030 772 9760 7461 6204 9098 -5566 -9183 -4651 -7748 -7917 4693 9410 -7927 8578 -6137 -1059 -1981 -8083 4977 -8317 -3793 6977 5003 -9432 2850 -2512 3808 -4504 -3186 -1342 -6716 6135 -5839 1660 -9080 -3606 -6339 -2169 6350 4497 -7782 5282 -1483 -2558 7363 -219 -3915 -3806 -6005 9805 2160 7494 -2009 6715 -6144 -6907 -7131 1706 -976 -2082 -6682 6149 -8817 6857 -5805 3950 2581 8112 -794 -7141 6772 -2310 -1715 -9328 -8833 5497 2978 2363 -9364 6015 9550 -4004 -9916 -8885 3404 9966 -3971 3724 -3266 8960 -5916 -4569 -3545 5829 656 -140 -3210 9690 -5387 6880 -2784 9860 3765 -8556 -4929 -9830 -5483 -1155 1411 712 -6571 5765 -580 7209 -9912 5727 -5400 -1865 7283 8699 6639 4353 -4431 799 -5551 8279 -3783 -9410 -2807 2420 6284 -897 -2316 -1113 6033 3741 5050 1489 1778 -9539 -9243 2517 -3440 -5414 -7804 6099 -8150 -4579 4104 5990 -2529 8791 -9986 8457 5805 8478 7373 -9035 -8521 -4587 -5819 -7491 -2685 2931 -5526 3593 4996 6321 -4082 3433 3491 717 715 -5783 1698 -8134 662 6181 -3935 176 8496 3094 292 -9707 7558 4421 -4697 9300 -3676 6639 1346 -3275 6153 -3291 7502 6778 -2546 -4769 -6728 -878 -1944 1046 -9386 950 -6204 -5973 -5425 4595 8697 7423 4870 -5784 6050 71 4498 -8973 -3174 -6253 -759 4931 6380 5742 -7362 2751 1400 5395 6435 6009 7554 7910 9759 1334 -4541 -215 -6641 7785 4597 1666 1999 -7060 9814 4351 -6788 1163 -9664 4381 5138 -4496 9570 9622 -9580 -8033 5866 -2332 8234 764 6897 2374 -2805 1358 -1038 -8355 -5359 3004 -9243 6360 -9757 7736 2336 -6502 2923 -4355 7128 -5364 7075 -5080 8349 4757 -9962 4705 646 -8284 -1103 -3360 -5882 -471 5940 3928 -376 8184 3020 -2203 5176 -379 -5374 -8584 9842 -4301 800 -4194 -9175 1670 2151 -1599 -7418 -9854 4993 -9571 -6051 6461 -8700 -3531 9134 3713 4149 3052 75 9629 -229 -295 426 -7906 -1776 -5569 5886 4803 -8226 -308 -2288 8048 3717 -4081 -6504 -5673 -5613 -2799 -3860 -1079 -6951 538 -2305 -104 -4723 -9702 1538 -5107 2981 -7825 -3063 2635 7335 -5256 -7468 3377 5104 6331 7090 6943 7170 9047 -6270 1882 -3584 4373 -5115 4011 3532 8439 3050 -7038 -7889 -7021 5063 2983 8602 4114 4563 -1871 8423 4346 3314 -459 -2660 -4808 5702 -5148 9347 -5742 -4956 564 7370 -2684 -3456 5215 4897 7709 -3185 7235 -9868 -7287 -3162 7230 1669 -1404 -5592 7454 6257 7883 -4392 1664 -5182 7217 4053 3527 78 -5653 -3525 7398 -708 -1431 1399 6541 6157 5477 -3841 9992 -2892 2606 -7454 -5138 -7568 8744 7219 -1795 -7383 -1531 -9118 