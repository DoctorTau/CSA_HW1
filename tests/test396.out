-328 -2362 -8246 -5294 -1842 -9162 -9918 -8010 -644 -2770 -7850 -9054 -5558 -3342 -8868 -1886 -9702 -474 -5318 -3116 -5110 -8508 -1010 -3112 -9536 -2468 -8062 -2030 -6920 -644 -7832 -7934 -5566 -7748 -9432 -2512 -4504 -3186 -1342 -6716 -9080 -3606 -7782 -2558 -3806 -6144 -976 -2082 -6682 -794 -2310 -9328 -9364 -4004 -9916 -3266 -5916 -140 -3210 -2784 -8556 -9830 -580 -9912 -5400 -9410 -2316 -3440 -5414 -7804 -8150 -9986 -5526 -4082 -8134 -3676 -2546 -6728 -878 -1944 -9386 -6204 -5784 -3174 -7362 -7060 -6788 -9664 -4496 -9580 -2332 -1038 -6502 -5364 -5080 -9962 -8284 -3360 -5882 -376 -5374 -8584 -4194 -7418 -9854 -8700 -7906 -1776 -8226 -308 -2288 -6504 -3860 -104 -9702 -5256 -7468 -6270 -3584 -7038 -2660 -4808 -5148 -5742 -4956 -2684 -3456 -9868 -3162 -1404 -5592 -4392 -5182 -708 -2892 -7454 -5138 -7568 -9118 